### PR TITLE
Add more print screenshots to chapter 11, and an example of border-radius rendering without clipping

### PR DESCRIPTION
**Pull request by chrishtr**

### CODE REVIEW

File: book/visual-effects.md [hunk 1]
```diff
@@ -623,6 +623,27 @@ class BlockLayout:
                 self.self_rect(), radius, bgcolor))
 ```
 
+::: {.web-only}
+With that, [this example](examples/example11-rounded-background.html) will round
+the corners of its background (notice that it does not round the text though:
```
Comment (pavpanchekha) [inline book/visual-effects.md:628]:
```suggestion
the corners of its background (notice that it does not round the text, though):
```

File: book/visual-effects.md [hunk 2]
```diff
@@ -623,6 +623,27 @@ class BlockLayout:
...
+
+::: {.print-only}
+With that, this example:
+
+::: {.transclude .html}
+www/examples/example11-rounded-background.html
+:::
+
+will round the corners of its background (notice that it does not round the text
+though:
```
Comment (pavpanchekha) [inline book/visual-effects.md:639]:
```suggestion
though):
```

File: book/visual-effects.md [hunk 3]
```diff
@@ -1436,9 +1459,28 @@ example:
...
+[longword]: examples/example11-longword.html
+
+![An example of overflowing text not being clipped by rounded corners \label{longword-example}](examples/example11-longword.png)
+
+:::
+
 ``` {.html .example}
 <div 
-  style="border-radius:30px;background-color:lightblue;overflow:clip">
+  style="\:30px;background-color:lightblue;overflow:clip">
```
Comment (pavpanchekha) [inline book/visual-effects.md:1483]: What?

**Review by pavpanchekha — APPROVED** [review]

File: book/visual-effects.md (hunk 3 above)
Comment (pavpanchekha) [reply book/visual-effects.md:1483]: I think this is a mistake

File: book/visual-effects.md [hunk 4]
```diff
@@ -623,6 +623,27 @@ class BlockLayout:
...
+:::
+
+::: {.print-only}
+With that, this example:
+
+::: {.transclude .html}
+www/examples/example11-rounded-background.html
+:::
+
+will round the corners of its background (notice that it does not round the text
```
Comment (chrishtr) [inline book/visual-effects.md:638]:
```suggestion
will round the corners of its background (notice that it does not round the text,
```

**Pull request merged**